README.md

```diff
@@ -18,18 +18,23 @@ NGSA Java App is inteneded for platform testing and monitoring in one or many Ku
 ```text
 
 Usage:
-        mvn clean spring-boot:run -Dspring-boot.run.arguments=[options] 
+        mvn clean spring-boot:run -Dspring-boot.run.arguments=[options]
+        
+        export [env var]=<value>
+        mvn clean spring-boot:run
 
 Options: 
-        --help                                                   Show help and usage information
-        --burst-header                                           Enable burst metrics header in healthz and version requests
-        --burst-service                                          Service name for bursting metrics (string) [default: ngsa-java]
-        --burst-target                                           Target level for bursting metrics (int) [default: 60]
-        --burst-max                                              Max level for bursting metrics (int) [default: 80]
-        --dry-run                                                Validates configuration
-        --log-level=<trace|info|warn|error|fatal>                Log Level [default: Error]
-        --version                                                Shows version information       
-
+        --help                                               Show help and usage information
+        --version                                            Shows version information
+        --dry-run                                            Validates configuration
+        --log-level=<trace|info|warn|error|fatal>            Log Level [default: Error]
+              
+Env vars:
+        BURST_HEADER=<true|false>                            Enable bursting metrics [default: false]
+        BURST_SERVICE                                        Service name for bursting metrics (string) [default: ngsa-java]
+        BURST_TARGET                                         Target level for bursting metrics (int) [default: 60]
+        BURST_MAX                                            Max level for bursting metrics (int) [default: 80]
+        PROMETHEUS=<true|false>                              Enable prometheus metrics [default: false]
 ```
```

Comment:
If both CLI and Env work then we need to keep the usage for CLI in.  Also, regarding the other thread, we should note that CLI args take precedence.